pkg/cloudprovider/provider/gce/types/types.go

```diff
@@ -44,6 +44,7 @@ type CloudProviderSpec struct {
 	MultiZone             providerconfigtypes.ConfigVarBool   `json:"multizone"`
 	Regional              providerconfigtypes.ConfigVarBool   `json:"regional"`
 	CustomImage           providerconfigtypes.ConfigVarString `json:"customImage,omitempty"`
+	PodNetworkFamily      providerconfigtypes.ConfigVarString `json:"podNetworkFamily,omitempty"`
```

Comment:
Apart from naming this just `NetworkFamily` instead of `PodNetworkFamily`, could we use some less generic type then string in this API?

Reply:
Not sure I understand what less generic means. Like int?

Reply:
I mean, wold it be possible to use the `NetworkFamily` typedef (or something similar) here?

Reply:
I guess not :) NVM